src/components/templates/Search/index.tsx

```diff
@@ -33,14 +34,13 @@ export default function SearchPage({
 
   useEffect(() => {
     if (!appConfig.metadataCacheUri) return
```

Comment:
If we don't pass `appConfig.metadataCacheUri` anymore we also don't need to check for it up here